src/kem/kyber/kem_kyber_1024.c

```diff
@@ -51,6 +57,16 @@ OQS_API OQS_STATUS OQS_KEM_kyber_1024_keypair(uint8_t *public_key, uint8_t *secr
 		return (OQS_STATUS) pqcrystals_kyber1024_ref_keypair(public_key, secret_key);
 	}
 #endif /* OQS_DIST_BUILD */
+#elif defined(OQS_ENABLE_KEM_kyber_1024_aarch64)
+#if defined(OQS_DIST_BUILD)
+	if () {
```

Comment:
Why is this `if ()` empty?

Reply:
A missed case in the template fragment. Thanks for catching it!